packaging/windows/Jenkinsfile.groovy

```diff
@@ -0,0 +1,72 @@
+  bat '''
+    if EXIST src\\github.com\\keybase\\client\\desktop\\release rmdir /q /s src\\github.com\\keybase\\client\\desktop\\release
+    path
+  '''
+  
+  stage 'Build Client'
+  bat '"%ProgramFiles(x86)%\\Microsoft Visual Studio 14.0\\vc\\bin\\vcvars32.bat" && src\\github.com\\keybase\\client\\packaging\\windows\\build_prerelease.cmd' 
+  stage 'Build UI'
+  bat 'src\\github.com\\keybase\\client\\packaging\\windows\\buildui.bat'
+  stage 'Build Installer'
+  bat 'call "%ProgramFiles(x86)%\\Microsoft Visual Studio 14.0\\vc\\bin\\vcvars32.bat" && src\\github.com\\keybase\\client\\packaging\\windows\\doinstaller_wix.cmd'
+  archiveArtifacts 'src\\github.com\\keybase\\client\\packaging\\windows\\${BUILD_TAG}\\*.*'
+
+  if (UpdateChannel != "None"){
+    stage 'Publish to S3'
+
+    step([$class: 'S3BucketPublisher', dontWaitForConcurrentBuildCompletion: false, entries: [[bucket: 'prerelease.keybase.io/windows', excludedFile: '', flatten: true, gzipFiles: false, keepForever: false, managedArtifacts: false, noUploadOnFailure: true, selectedRegion: 'us-east-1', showDirectlyInBrowser: false, sourceFile: 'src\\github.com\\keybase\\client\\packaging\\windows\\${BUILD_TAG}\\*.exe', storageClass: 'STANDARD', uploadFromSlave: true, useServerSideEncryption: false]], profileName: 'keybase', userMetadata: []])
+    step([$class: 'S3BucketPublisher', dontWaitForConcurrentBuildCompletion: false, entries: [[bucket: 'prerelease.keybase.io', excludedFile: '', flatten: true, gzipFiles: false, keepForever: false, managedArtifacts: false, noUploadOnFailure: true, selectedRegion: 'us-east-1', showDirectlyInBrowser: false, sourceFile: 'src\\github.com\\keybase\\client\\packaging\\windows\\${BUILD_TAG}\\update-windows-prod-test-v2.json', storageClass: 'STANDARD', uploadFromSlave: true, useServerSideEncryption: false]], profileName: 'keybase', userMetadata: []])
+  }
+
+  if (UpdateChannel == "Smoke"){
+    def clientCommit = getCommit('src\\github.com\\keybase\\client')
+    def kbfsCommit =  getCommit('src\\github.com\\keybase\\kbfs')
+    def updaterCommit =  getCommit('src\\github.com\\keybase\\go-updater')
+    def releaseCommit =  getCommit('src\\github.com\\keybase\\release')
+    def smokeASemVer = ''
+    dir('src\\github.com\\keybase\\client\\go\\keybase') {
+        smokeASemVer = bat(returnStdout: true, script: '@echo off && winresource.exe -cv').trim()
+    }
+    build job: 'windows-installer-pipeline', parameters: [string(name: 'ClientRevision', value: "${clientCommit}"), string(name: 'KBFSRevision', value: "${kbfsCommit}"), string(name: 'UpdaterRevision', value: "${updaterCommit}"), string(name: 'ReleaseRevision', value: "${releaseCommit}"), string(name: 'DOKAN_PATH', value: "${DOKAN_PATH}"), string(name: 'UpdateChannel', value: 'Smoke2'), string(name: 'SmokeASemVer', value: "${smokeASemVer}")], wait: false
```

Comment:
The actual command needs to be invoked with "1.0.18-20161127110747+72b1d3f" in place of "${SmokeASemVer}", etc. but it doesn't look like the substitution is happening.